src/bindings/js/node/src/core_wrap.cpp

```diff
@@ -232,6 +233,31 @@ Napi::Value CoreWrap::get_available_devices(const Napi::CallbackInfo& info) {
     return js_devices;
 }
 
+Napi::Value CoreWrap::get_versions(const Napi::CallbackInfo& info) {
+    if (info.Length() == 0) {
+        reportError(info.Env(), "No argument provided in the getVersions() method call.");
+        return info.Env().Undefined();
+    }
+    auto device_arg = info[0];
+    if (!device_arg.IsString()) {
+        reportError(info.Env(), "The argument in getVersions() method must be a string or convertible to a string.");
+        return info.Env().Undefined();
+    }
+    const auto& devices_map = _core.get_versions(device_arg.ToString());
+    Napi::Object versionsObject = Napi::Object::New(info.Env());
```

Comment:
Better to use snake case notation at c++ side
```suggestion
    Napi::Object versions_object = Napi::Object::New(info.Env());
```